docs/it/index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Casa
```

Comment:
"Home" cannot be translated in "Casa"... please leave "home" untranslated. 
Home is commonly used even in italian.